lending_pool/tests/rust_tests.rs

```diff
@@ -270,36 +271,36 @@ fn repay_test() {
         &user_addr,
         BORROW_USDC_TOKEN_ID,
         1,
-        &rust_biguint!(500),
+        &rust_biguint!(1_000 * BP),
         Some(&Vec::<u8>::new()),
     );
 
     lending_setup.b_mock.check_nft_balance(
         &user_addr,
         USDC_TOKEN_ID,
         0,
-        &rust_biguint!(500),
+        &rust_biguint!(1_000 * BP),
         Some(&Vec::<u8>::new()),
     );
 
-    lending_setup.b_mock.set_block_timestamp(200_000);
+    lending_setup.b_mock.set_block_round(12);
 
     lending_setup
         .b_mock
-        .set_esdt_balance(&user_addr, USDC_TOKEN_ID, &rust_biguint!(503));
+        .set_esdt_balance(&user_addr, USDC_TOKEN_ID, &rust_biguint!(1_025 * BP));
```

Comment:
This is Scenario 1 from [here](https://docs.google.com/spreadsheets/d/1G-6mqcFT4LQizkvb2A2_b_DaEr3M7TzZpJSXkxVxWac/edit#gid=1070180197)